nmtwizard/preprocess/consumer.py

```diff
@@ -111,25 +111,25 @@ def __init__(self, config, result_dir, tok_step):
 
     def __call__(self, tu_batch):
 
-        # TODO V2 : feed tokenized words ?
         # TODO : remove value for placeholders
         for tu in tu_batch :
             if 'source' in self._vocabularies:
-                for token in itertools.chain.from_iterable(tu.get_src_tok()):
+                for token in itertools.chain.from_iterable(tu.src_tok[1]):
```

Comment:
It a bit unclear why we need to index `src_tok` with 1. Maybe add a getter `tu.src_tokens` that directly return a list of tokens as expected.